packages/privacy-toolset/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/privacy-toolset",
-	"version": "1.0.1",
+	"version": "1.0.0",
```

Comment:
Why are we moving back a version? Apologies if I'm missing something obvious

Reply:
Sorry for the confusion. As [mentioned](https://github.com/Automattic/wp-calypso/pull/70601#discussion_r1038212556), I've decided to move the changes to @automattic/privacy-toolset to a separate branch (https://github.com/Automattic/wp-calypso/pull/70709) that has now been merged and published on NPM